modules/virt-enabling-preallocation-globally.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-using-preallocation-for-datavolumes.adoc
+
+[id="virt-enabling-preallocation-globally_{context}"]
+= Enabling preallocation globally
+
+You can enable preallocation mode for the Containerized Data Importer (CDI) for the whole cluster by adding the `prellocation` field to the `CDI` object.
```

Comment:
Change:
You can enable preallocation mode for the Containerized Data Importer (CDI) for the whole cluster by adding the `prellocation` field to the `CDI` object.
To:
You can enable cluster wide preallocation mode for the Containerized Data Importer (CDI), by adding the `prellocation` field to the `CDI` object.